src/node.cc

```diff
@@ -1923,6 +1923,63 @@ static void Uptime(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+static void GetResourceUsage(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  uv_rusage_t rusage;
+  int err = uv_getrusage(&rusage);
+  if (err)
+    return env->ThrowUVException(err, "uv_getrusage");
+
+  Local<Object> resource_usage = Object::New(env->isolate());
```

Comment:
@thlorenz @bnoordhuis Usually the first iteration of this ```process.resourceUsage``` is the slowest with 0.042276ms, but other iterations take ~0.019789ms - the lowest of my iterations was around ~0.011ms. Compared to the first ```os.loadavg()``` (just as an example for a function that calls every time the libuv function) call with 0.026139ms other calls are around ~0.009892ms. All numbers are measured using the js functions.